doc/api.md

```diff
@@ -224,7 +225,9 @@ the API resource fields and the same fields in the database model.
 | `internal_attributes` | `internalAttributes` | List of internal attributes with free format for specific IoT Agent configuration                                                                                                                                                                                         | LWM2M mappings from object URIs to attributes |
 | `static_attributes`   | `staticAttributes`   | List of static attributes to append to the entity. All the updateContext requests to the CB will have this set of attributes appended.                                                                                                                                    | `[ { "name": "attr_name", "type": "Text" } ]` |
 | `expressionLanguage`  | `expresionLanguage`  | optional boolean value, to set expression language used to compute expressions, possible values are: legacy or jexl. When not set or wrongly set, legacy is used as default value.                                                                                        |
-| `explicitAttrs`       | `explicitAttrs`      | Boolean value to support selective ignore of measures for device so that IOTA doesn’t progress. If not specified default is false.                                                                                                                                        | `true/false`                                  |
+
+| `explicitAttrs`       | `explicitAttrs`      | Boolean value to support selective ignore of measures for device so that IOTA doesn’t progress. If not specified default is `false`.                                                                                                                                      | `true/false`                                  |
+| `ngsiVersion`         | `ngsiVersion`        | optional string value used in mixed mode to switch between **NGSI-v2** and **NGSI-LD**payloads. The default is `v2`.                                                                                                                                                      | `v2/ld/mixed`                                       |
```

Comment:
It seems markdown rendering is broken here.

![imagen](https://user-images.githubusercontent.com/1534240/100593614-dadb1080-32f8-11eb-96b0-3e6804f5459d.png)

Reply:
Fixed : 9ad9268